src/platform/forms/components/FormFooter.jsx

```diff
@@ -1,19 +1,31 @@
 import React from 'react';
 
-import AskVAQuestions from './AskVAQuestions';
-
 export default class FormFooter extends React.Component {
   render() {
     const { formConfig, currentLocation } = this.props;
     const GetFormHelp = formConfig.getHelp;
-    const trimmedPathname = currentLocation.pathname.replace(/\/$/, '');
-    const isConfirmationPage = trimmedPathname.endsWith('confirmation');
+
+    if (
+      currentLocation &&
+      currentLocation.pathname.replace(/\/$/, '').endsWith('confirmation')
+    ) {
+      return null;
+    }
+
+    if (!GetFormHelp) {
```

Comment:
Should we make `getHelp` a required prop?

Reply:
Probably, I was trying to be extra cautious here.